nncf/common/quantization/quantizer_removal.py

```diff
@@ -62,6 +62,17 @@ def _parse_node_relatives(node: NNCFNode, is_parents: bool):
                     if relative not in to_cut:
                         to_cut.append(relative)
                     to_see_children.append(relative)
+                    # We should see parents for the `relative` node here only if they are
```

Comment:
Do you plan to add a test on this functionality for ONNX?

Reply:
This test is already present. See tests/common/quantization/test_quantizer_removal.py